migrations/versions/0298_add_mou_signed_receipt.py

```diff
@@ -0,0 +1,146 @@
+"""
+
+Revision ID: 0298_add_mou_signed_receipt
+Revises: 0297_template_redacted_fix
+Create Date: 2019-05-22 16:58:52.929661
+
+"""
+from alembic import op
+from flask import current_app
+
+
+revision = '0298_add_mou_signed_receipt'
+down_revision = '0297_template_redacted_fix'
+
+
+templates = [
+    {
+        'id': '4fd2e43c-309b-4e50-8fb8-1955852d9d71',
+        'name': 'MOU Signed By Receipt',
+        'type': 'email',
+        'subject': 'You’ve accepted the GOV.​UK Notify data sharing and financial agreement',
+        'content_lines': [
+            'Hi ((signed_by_name)),',
+            '',
+            '((org_name)) has accepted the GOV.​UK Notify data sharing and financial agreement. ',
+            '',
+            'If you need another copy of the agreement you can download it here: ((mou_link))',
+            '',
+            'If you need to add Cabinet Office as a supplier, here are the details you need:',
+            '',
+            'TO BE ADDED MANUALLY',
```

Comment:
Does this need to be updated?

Reply:
yes, but through the admin interface so that we don't do anything. ah. hmm.

i'll split out the migration into a separate PR so we can do that before the automatic sending happens

Reply:
I see the templates are in prod - is someone fixing the content?

Reply:
pete fixed the content just now